src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/utils/UpdateSessionTemplateValidator.kt

```diff
@@ -121,4 +141,49 @@ class UpdateSessionTemplateValidator(
 
     return errorMessages
   }
+
+  private fun validateUpdateSessionLocation(existingSessionTemplate: SessionTemplateDto, updateSessionDetails: SessionDetailsDto, hasFutureBookedVisits: Boolean): String? {
+    // if a session has booked visits all locations should be accommodated post update
+    val errorMessage = "Cannot update locations to the new location list as all existing locations in session template are not catered for."
```

Comment:
Make this a constant or put it in the after the:
return "Cannot update...

Reply:
Updated.